mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java

```diff
@@ -147,21 +148,22 @@ private static void executeSolrCommand(String coreID, String command) throws Uns
         HttpPost post = new HttpPost(coreURL + "/config");
         post.setHeader("Content-type", "application/json");
         post.setEntity(new StringEntity(command));
+        String commandprefix = command.indexOf('-') != -1 ? command.substring(2,command.indexOf('-')) : "unknown command";
         HttpResponse response;
         try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
             response = httpClient.execute(post);
             String respContent = new String(ByteStreams.toByteArray(response.getEntity().getContent()),
                 StandardCharsets.UTF_8);
             if (response.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
-                LOGGER.debug("SOLR config update command was successful \n" + respContent);
+                LOGGER.debug("SOLR config " + commandprefix + " command was successful \n" + respContent);
```

Comment:
You could use:
`LOGGER.debug("SOLR config {} command was successful \n {}",commandprefix, respContent);`

Reply:
thank you